streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -1198,6 +1200,7 @@ private static <K, V> KafkaConsumer<K, V> createConsumer(final Properties consum
         filtered.putAll(consumerConfig);
         filtered.setProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
         filtered.setProperty(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "true");
+        filtered.setProperty(ConsumerConfig.GROUP_PROTOCOL_CONFIG, GroupProtocol.CLASSIC.name);
```

Comment:
this is not used by the resetter tool, but it's needed to ensure resetters tests like `testReprocessingFromScratchAfterResetWithIntermediateUserTopic` pass without relying on the default protocol (they use another consumer to "waitUntilMinKeyValueRecordsReceived").